molgenis-core-ui/src/main/resources/templates/molgenis-header.ftl

```diff
@@ -137,8 +137,27 @@
     </#if>
 
 <#-- Start application content -->
-<div class="container-fluid mg-page-content"
-     style="margin-top: <#if app_settings.logoTopHref?? && (!version?? ||version == 1)>${app_settings.logoTopMaxHeight + 50}<#else>50</#if>px;">
+<#if (!version?? ||version == 1)>
+    <script>
+        // Fix bootstrap 3 menu when the menu gets to big and needs 2 lines.
+        function calcHeaderHeight(){
+            let pageContext = document.getElementsByClassName("mg-page-content")[0];
+            let navBar = document.getElementsByClassName("navbar-fixed-top")[0];
+            let height = 50;
+
+            <#if app_settings.logoTopHref?has_content>
+                let logoTopHeight = ${app_settings.logoTopMaxHeight};
+                if(logoTopHref) height += logoTopHeight; // Correct height for header image
+            </#if>
+            
+            if(navBar.getBoundingClientRect().height>50) height += 50; // Correct height for double sized menu
+            pageContext.style.marginTop = height + "px";
+        }
+        window.addEventListener('resize', calcHeaderHeight);
```

Comment:
A good practise is to remove the event listener before adding it, to prevent multiple eventlisteners to be added for the same function (memory leak hazard)